test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -392,7 +392,8 @@ def setUp(self):
         Job.dflt_attributes[ATTR_k] = 'oe'
         # Increase the log level
         a = {'log_events': '2047'}
-        self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+        rv = self.server.manager(MGR_CMD_SET, SERVER, a, expect=True)
+        self.assertEqual(rv, 0)
```

Comment:
No need to check the return code of manager().  It will raise an exception if it fails (same fore the rest of the tests).